chain/client/src/client.rs

```diff
@@ -1105,30 +1108,20 @@ impl Client {
             };
             self.chain.blocks_with_missing_chunks.prune_blocks_below_height(last_finalized_height);
 
-            let now = Clock::instant();
-            let result = if self.config.archive {
-                self.chain.clear_archive_data(self.config.gc.gc_blocks_limit)
-            } else {
-                let tries = self.runtime_adapter.get_tries();
-                self.chain.clear_data(tries, &self.config.gc)
-            };
-            if let Err(err) = result {
-                error!(target: "client", "Can't clear old data, {:?}", err);
-                debug_assert!(false);
-            };
-            let gc_time = now.elapsed();
-            metrics::GC_TIME.observe(gc_time.as_secs_f64());
-            // it's safe to unwrap here because block is already accepted
-            if block.header().height()
-                - self.runtime_adapter.get_epoch_start_height(block.hash()).unwrap()
-                < EPOCH_START_INFO_BLOCKS
             {
-                info!(
-                    "spent {:?} on gc after processing block {:?} at height {}",
-                    gc_time,
-                    block.hash(),
-                    block.header().height()
-                );
+                let _span = info_span!(target: "client", "GC collection", block_hash=?block.hash(), height=block.header().height()).entered();
```

Comment:
Nit: GC already stands for garbage collection

```suggestion
                let _span = info_span!(target: "client", "Garbage collection", block_hash=?block.hash(), height=block.header().height()).entered();
```